src/chart/pie/PieView.ts

```diff
@@ -242,6 +243,29 @@ class PieView extends ChartView {
             }
         }
 
+        // remove empty-circle(which is not instance of PiePiece) if it exists
+        group.children().forEach(child => {
+            if (!(child instanceof PiePiece)) {
```

Comment:
It's not a good way to check if it's the empty-circle element by checking it's not a piece of the pie. It may have issues if we add other elements that are not a piece of the pie further.

A better way is to save this empty-circle element as a private property of PieView. It's simpler and more robust.

Reply:
fixed